pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -368,9 +375,8 @@ func (e *etcdObserver) setBootstrapMember() error {
 	for _, subset := range endpoints.Subsets {
 		for _, address := range subset.Addresses {
 			if address.Hostname == "etcd-bootstrap" {
-				ip := address.IP
-				peerURLs := fmt.Sprintf("https://%s:2380", ip)
-				clusterMember, err := setMember(ip, []string{peerURLs}, ceoapi.MemberUnknown)
+				peerURLs := fmt.Sprintf("https://%s:2380", address.IP)
```

Comment:
do we check if IP has value?

Reply:
I think we check in hostendpoint controller quite exhaustively if IP is present. Will add a check to be extra safe like in KAO

Reply:
thanks

Reply:
added